service/service.go

```diff
@@ -105,8 +107,7 @@ func (svc *Service) Close(ctx context.Context) error {
 	return nil
 }
 
-func (svc *Service) registerCheckers(ctx context.Context) (err error) {
-
+func (svc *Service) registerCheckers(ctx context.Context) error {
 	if svc.cantabularClient != nil {
 		if svc.Config.CantabularHealthcheckEnabled {
```

Comment:
dataset checker should probably be added here? I added it so probs just can wait for that